test/circleci/rollcage/test_throwables.clj

```diff
@@ -0,0 +1,17 @@
+(ns circleci.rollcage.test-ring-middleware
```

Comment:
Should be `circleci.rollcage.test-throwables`

Reply:
This explains why my REPL was being so strange.